salt/utils/network.py

```diff
@@ -1846,8 +1846,16 @@ def refresh_dns():
         pass
 
 
+@jinja_filter('connection_check')
+def connection_check(addr, port=80, safe=False, ipv6=None):
+    '''
+    Provides a convenient alias for the dns_check filter.
+    '''
+    return dns_check(addr, port, safe, ipv6)
+
+
 @jinja_filter('dns_check')
-def dns_check(addr, port, safe=False, ipv6=None):
+def dns_check(addr, port=80, safe=False, ipv6=None):
```

Comment:
What's the reasoning behind defaulting this to port 80?

Reply:
Cool. That makes sense. This shouldn't cause any change in current behavior so I think we're fine here. Thanks, @MTecknology